### PR TITLE
Test-only PR with every event

### PR DESCRIPTION
**Pull request by fregante**
For #4029 

Please don't touch

### CODE REVIEW

**Comment by fregante** [comment]
Comment

**Comment by fregante** [comment]
Hidden comment

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// MUCH IGNORE
```
Comment (fregante) [inline .gitignore:1]: Review comment

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 node_modules
```
Comment (fregante) [inline .gitignore:2]: Resolved review comment

File: .gitignore (hunk 1 above)
Comment (fregante) [reply .gitignore:1]: Resolved single review comment

**Review by fregante — COMMENTED** [review]
Review

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 yarn.lock
```
Comment (fregante) [inline .gitignore:3]: Comment part of a review

**Review by fregante — COMMENTED** [review]
Review with comment and this comment

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
...
 distribution/build
```
Comment (fregante) [inline .gitignore:4]: Comment part of a review without main comment

File: .gitignore (hunk 1 above)
Comment (fregante) [reply .gitignore:1]: Comment part of a whole review started later

File: .gitignore (hunk 3 above)
Comment (fregante) [reply .gitignore:3]: Comment part of a whole review started later, but also resolved

File: .gitignore [hunk 5]
```diff
@@ -1,7 +1,9 @@
...
 test/web-ext-profile
 !test/web-ext-profile/.gitkeep
 .cache
 .parcel-cache
+// VERY NOISE
```
Comment (fregante) [inline .gitignore:9]: Outdated comment

**Comment by fregante** [comment]
dat's alot.

File: .gitignore (hunk 2 above)
Comment (fregante) [reply .gitignore:2]: Comment part of a whole review started later, but also resolved

**Review by fregante — COMMENTED** [review]
Whole review, with comment in past, resolved review

**Comment by fregante** [comment]
@yakov116 can you leave a no-comment Approve review? Reopen and close if necessary

File: .gitignore [hunk 6]
```diff
@@ -1,7 +1,9 @@
+// MUCH IGNORE
 node_modules
 yarn.lock
 distribution/build
 test/web-ext-profile
 !test/web-ext-profile/.gitkeep
 .cache
 .parcel-cache
```
Comment (fregante) [inline .gitignore:8]: Resolved comment in unresolved review thread

File: .gitignore [hunk 7]
```diff
@@ -1,7 +1,9 @@
+// MUCH IGNORE
 node_modules
 yarn.lock
 distribution/build
 test/web-ext-profile
 !test/web-ext-profile/.gitkeep
```
Comment (fregante) [inline .gitignore:6]: Review with multiple threads (thread 1)

File: .gitignore [hunk 8]
```diff
@@ -1,7 +1,9 @@
...
 .cache
```
Comment (fregante) [inline .gitignore:7]: Review with multiple threads (thread 2)

**Review by yakov116 — CHANGES_REQUESTED** [review]
Test request changes

**Review by yakov116 — APPROVED** [review]

**Review by yakov116 — APPROVED** [review]
Appove with comment

**Comment by fregante** [comment]
Someone please delete this comment so we get the "Comment deleted" event

**Pull request closed without merging**